CONTRIBUTING.md

```diff
@@ -4,4 +4,97 @@ Thank you for your interest in the Langium project! The following is a set of gu
+
+#### Altered Langium for own language projects
+
+There is an npm build target available (`npm run dev-build`) linking all Langium packages to your global scope. It unlinks and uninstalls the Langium packages from the global scope, deletes any `node_modules` folders below the packages directory (see warning above), afterwards performs `npm install` and then links all packages to the global scope again. Then your are able to use `yo langium` or `langium generate` containing your local Langium adjustments from everywhere with your local user.
+
+A project you created with `yo langium` contains a dependency to `langium` (e.g. `0.1.0`) and a dev-dependency to `langium-cli`) inside `package.json` by default. Now, the easiest way to switch to your own build is to locally reference the required Langium packages folders.
+Let's assume your local Langium checkout resides on the same level as your own language project, then these two commands will set the references inside the `package.json`:
+```shell
+npm install --save ../langium/packages/langium
+npm install --save-dev ../langium/packages/langium-cli
+```
+
+The first command above will for example change the dependency entry for `langium` from
+```json
+"langium": "0.1.0"
+```
+to
+```json
+"langium": "file:../langium/packages/langium"
+```
+inside `package.json`.
+
+#### Use VSCode workspace to simplify debugging
+
+When debugging a project that depends on another project under development using a `code-workspace` makes life easier. Open an empty VSCode instance and *"Add Folder to Workspace..."* and add the Langium directory and your language project directory to the workspace. Save it with *"Save Workspace As..."*. It is recommended to have your language project and Langium itself reside on the same level on the same directory level.
+
+**Reminder:** Don't forget to run `npm run watch` in your language project and the Langium package or any other package where you want to change code and see the effects when executing and debugging.
+
+In `.vscode/launch.json` of your language project add the following launch configuration:
+```json
+        {
+			"name": "Attach",
+			"port": 6009,
+			"request": "attach",
+			"skipFiles": [
+				"<node_internals>/**"
+			],
+			"sourceMaps": true,
+			"outFiles": [
+				"${workspaceFolder}/out/**/*.js"
+			],
+			"type": "pwa-node"
+		}
+```
+With the above configuration you can attach the debugger to the language server which effectively means you are able to debug your local version of Langium.
+
+**Hint:** If you want to halt the language server start until you attach the debugger you have to `--inspect=6009` to `--inspect-brk=6009` in the language project's `src/extension.ts` like shown below:
```

Comment:
```suggestion
**Hint:** If you want to halt the language server start until you attach the debugger you have to change `--inspect=6009` to `--inspect-brk=6009` in the language project's `src/extension.ts` like shown below:
```

Reply:
We could control this via an environment variable so a code change is not necessary. Could you create an issue for that?

Reply:
I will change it accordingly and I just created #215